sdk/servicebus/Azure.Messaging.ServiceBus/src/ServiceBusException.cs

```diff
@@ -189,10 +118,10 @@ public enum FailureReason
             ClientClosed,
 
             /// <summary>A client was forcefully disconnected from an Service Bus entity instance.</summary>
-            ConsumerDisconnected,
+            ReceiverDisconnected,
 
             /// <summary>An Service Bus resource, such as an Service Bus entity, consumer group, or partition cannot be found by the Service Bus service.</summary>
```

Comment:
This comment is not valid for Service Bus.

Reply:
Fixed